### ISSUE TITLE
Topics to Discuss on Monday 22nd June

### ISSUE THREAD

**Issue by NavonilNeogi** [A-F-V/T3M#12]
1) What is a Machine Learning

2) What is special about Machine Learning and why do we do it?

3) Gradient descent and Cost Functions

4) Linear and Logistic Regression

Random Forests

Forwardpropagation and backpropagation (maybe not required, as these are neural networks)

**Comment by A-F-V** [comment]
Maybe we do random forests and decision trees for next week

**Comment by hsuy141** [comment]
Notebooks on regression and decision trees uploaded under folder 'Theory Notebooks'

**Comment by hsuy141** [comment]
Suggestion:
a) Introduction to Machine Learning; Rudimentary Statistical Learning Theory
b) Linear Regression
c) Classification (Logistic Regression)
d) Decision Trees and Random Forests for Regression and Classification
e) Gradient Boosting algorithms
f) Support Vector Machines
g) Feedforward Neural Networks
h) Convolutional Neural Networks

**Comment by A-F-V** [comment]
When do you think we should aim to discuss each of these? Should we say a) for Monday?

**Comment by hsuy141** [comment]
> When do you think we should aim to discuss each of these? Should we say a) for Monday?

Do we mean later today or in 7 days time?

````

**Comment by A-F-V** [comment]
I though originally for later today, but that’s tight

**Comment by NavonilNeogi** [comment]
I'm good for today
